### PR TITLE
Added support for Ubuntu 20.04 in iCubOS

### PR DESCRIPTION
**Pull request by mbrunettini**

### CODE REVIEW

File: docs/icub_operating_systems/icubos/installation-from-scratch.md [hunk 1]
```diff
@@ -20,11 +20,36 @@ Follow the below steps to install Ubuntu server on icub-head.
 
 Please note that the installation procedure can be slight different in each release, you can see the [Ubuntu server official install guide](https://ubuntu.com/tutorials/tutorial-install-ubuntu-server)
 
+## Ubuntu Installer configuration_
```
Comment (pattacini) [inline docs/icub_operating_systems/icubos/installation-from-scratch.md:23]:
```suggestion
## Ubuntu Installer configuration
```

**Review by pattacini — CHANGES_REQUESTED** [review]

**Review by pattacini — APPROVED** [review]

**Review by pattacini — APPROVED** [review]
Nice!

Let's remember to squash and merge from the GH PR button.

File: docs/icub_operating_systems/icubos/installation-from-scratch.md [hunk 2]
```diff
@@ -162,7 +196,7 @@ To use the IMU Bosch BNO055 through the i2c bus these additional steps are requi
 
 ### Test the configuration of i2c and BNO055
 
-To test the installation with the python script BNO055:
+To test the installation with the [python script BNO055](https://github.com/icub-tech-iit/icub-os-files/blob/master/drivers/imu-bosh/BNO055.zip):
```
Comment (traversaro) [inline docs/icub_operating_systems/icubos/installation-from-scratch.md:199]: It was a typo that was there also before, but just to point it out (we can also fix it in a later PR): the IMU is manufactured from Bos*c*h .

File: docs/icub_operating_systems/other-machines/generic-machine.md [hunk 3]
```diff
@@ -258,12 +177,34 @@ Using the gnome3 control panel, it is not possible to avoid the system has to be
...
+2. Install the appropriate nvidia driver, as follows (for the example above)
+
+`sudo apt install nvidia-331`
 
-` nvidia-glx nvidia-settings nvidia-xconfig`
+3. Reboot
 
-- Run nvidia-xconfig
-- Reboot
+For further details, please read [thi guide](https://help.ubuntu.com/community/BinaryDriverHowto/Nvidia)
```
Comment (traversaro) [inline docs/icub_operating_systems/other-machines/generic-machine.md:207]:
```suggestion
For further details, please read [this guide](https://help.ubuntu.com/community/BinaryDriverHowto/Nvidia)
```

**Review by traversaro — APPROVED** [review]
Minor comments.

**Comment by pattacini** [comment]
I'll fix them straight in master.

**Comment by pattacini** [comment]
Done ✔

**Comment by mbrunettini** [comment]
links seems ok now

**Comment by mbrunettini** [comment]
Actually it was not a typo, but the [destination path](https://github.com/icub-tech-iit/icub-os-files/tree/master/drivers/imu-bosch) was wrong, fixed.

**Pull request merged**